src/Common/src/System/Net/SocketAddressPal.Unix.cs

```diff
@@ -65,7 +65,7 @@ public static unsafe AddressFamily GetAddressFamily(byte[] buffer)
             return family;
         }
 
-        public static unsafe void SetAddressFamily(byte[] buffer, AddressFamily family)
+        public static unsafe void SetAddressFamily(Span<byte> buffer, AddressFamily family)
         {
             Interop.Error err;
             fixed (byte* rawAddress = buffer)
```

Comment:
I can revert this because this is no longer used (it moved to pal_networking.c).

> Nit: can we change buffer to &MemoryMarshal.GetReference(buffer)?

For my education, why/what is the difference?

Reply:
> For my education, why/what is the difference?

It's very minor, but I talk about it in https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-core-3-0/... search for GetPinnableReference.

Reply:
Interesting blog post! And while reading, surprised and proud to get a mention.